pkg/build/controller/image_change_controller.go

```diff
@@ -17,6 +17,8 @@ import (
 	imageapi "github.com/openshift/origin/pkg/image/api"
 )
 
+var BuildConfigInstantiateError = fmt.Errorf("not all build configurations processed successfully, will retry")
```

Comment:
Use `errors.new`. The error message could probably be a bit more descriptive, maybe something like: `an error occurred processing 1 or more build configurations; the image change trigger for image stream $ns/$stream:$tag will be retried`